composer.json

```diff
@@ -65,10 +64,7 @@
 		]
 	},
   "config": {
-      "allow-plugins": {
-          "dealerdirect/phpcodesniffer-composer-installer": true
```

Comment:
@markkelnar any reason this is being removed?

Reply:
Looks like the history from WPGraphQL core was: https://github.com/wp-graphql/wp-graphql/pull/2291#issue-1165612063

Reply:
Just want to make sure we're not causing problems with this removal

Reply:
I was spending time on the other codesniff rules. Didn't finish that work yet so decided to back out those plugins. This must have been removed by composer in that cleanup. I'll see if anything depends on it.

Reply:
@markkelnar possibly this failing test is related? https://github.com/wp-graphql/wp-graphql-smart-cache/actions/runs/5659860880/job/15334309544#step:7:14